LiskHQ/lisk-nano
# Refactor i18n - Closes #892

### What was the problem?
We had 2 different setups of i18n (one for electron, one for the web version) that could share one setup file

### How did I fix it?
Put language related things into a directory 'i18n' that can be shared

### How to test it?
- Run tests
- Make sure the translations are still working

### Review checklist
- [x] The PR solves https://github.com/LiskHQ/lisk-nano/issues/892
